src/main/java/com/ceos19/springeverytime/domain/auth/jwt/LoginFilter.java

```diff
@@ -0,0 +1,55 @@
+package com.ceos19.springeverytime.domain.auth.jwt;
+
+import com.ceos19.springeverytime.domain.user.dto.response.CustomUserDetails;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+import java.io.IOException;
+
+
+@RequiredArgsConstructor
+public class LoginFilter extends UsernamePasswordAuthenticationFilter {
+
+    private final AuthenticationManager authenticationManager;
+    private final JWTUtil jwtUtil;
+
+    @Override
+    public Authentication attemptAuthentication(HttpServletRequest request, HttpServletResponse response) throws AuthenticationException {
+        String username = obtainUsername(request);
+        String password = obtainPassword(request);
+
+        UsernamePasswordAuthenticationToken authToken
+                = new UsernamePasswordAuthenticationToken(username, password, null);
```

Comment:
UsernamePasswordAuthenticationToken authToken
```suggestion
        UsernamePasswordAuthenticationToken authToken
                = new UsernamePasswordAuthenticationToken(username, password);
```

`UsernamePasswordAuthenticationToken`을 보면 생성자 두 개가 있습니다. 찬님은 authorities에 null을 넣으셨는데, null을 넣어주지 않아도 username과 password로만 `UsernamePasswordAuthenticationToken`을 생성하면 authorities에는 null이 들어갑니다!

```java
public UsernamePasswordAuthenticationToken(Object principal, Object credentials) {
    super((Collection)null);
    this.principal = principal;
    this.credentials = credentials;
    this.setAuthenticated(false);
}
```

Reply:
헉 몰랐던 내용이네요..! 감사합니다!